guides/source/configuring.md

```diff
@@ -1109,6 +1109,13 @@ text/javascript image/svg+xml application/postscript application/x-shockwave-fla
     1. `select=eq(n\,0)+eq(key\,1)+gt(scene\,0.015)`: Select the first video frame, plus keyframes, plus frames that meet the scene change threshold.
     2. `loop=loop=-1:size=2,trim=start_frame=1`: To use the first video frame as a fallback when no other frames meet the criteria, loop the first (one or) two selected frames, then drop the first looped frame.
 
+* `config.active_storage.transformations_for_format` can be used to specify transformations that should be applied to all images of a given content_type.
+
+    ```ruby
+    config.active_storage.transformations_for_format[:jpg] = { saver: { quality: 80, strip: true } }
+    config.active_storage.transformations_for_format[:png] = { saver: { compression: 9, strip: true } }
```

Comment:
I find the `for` somewhat tough to wrap my head around both on the configuration side and the implementation side. Let's call this `transformations_by_format`.